build.ps1

```diff
@@ -460,11 +460,31 @@ function RestartAdminApiContainer {
 }
 
 function BuildAdminApiDevDockerImage {
-    &docker build -t adminapi-dev --no-cache -f "$dockerRoot/dev.pgsql.Dockerfile" .
+    Push-Location $dockerRoot
+    try {
+        ">>> Building dev.pgsql.Dockerfile" | Out-Host
+        &docker build `
+            -t adminapi-dev-pgsql `
+            --build-context assets=$(Resolve-Path "..") `
+            --no-cache `
+            -f "dev.pgsql.Dockerfile" `
+            .
+
+        ">>> Building dev.mssql.Dockerfile" | Out-Host
+        &docker build `
+            -t adminapi-dev-mssql `
+            --build-context assets=$(Resolve-Path "..") `
+            --no-cache `
+            -f "dev.mssql.Dockerfile" `
+            .
```

Comment:
Although line 487 below only runs the pgsql image, it seems valuable to _build_ the MSSQL image here as well, so that we will know if there are any errors.

Reply:
I don't think anyone is really using this script. At some point yes it would make sense to have an easy-to use command here for running in MSSQL mode.